src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -86,39 +95,54 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
   width: $range-thumb-size;
   height: $range-thumb-size;
   border-radius: 50%;
-  border: border-width() solid color('sky', 'lighter');
+  border: border-width() solid
+    var(--p-action-interactive, color('sky', 'lighter'));
   box-shadow: $range-thumb-shadow;
-  background: linear-gradient(color('sky', 'lighter'), color('sky', 'light'));
+  background: linear-gradient(
+    var(--p-action-interactive, color('sky', 'lighter')),
+    var(--p-action-interactive, color('sky', 'light'))
+  );
   -webkit-tap-highlight-color: transparent;
 
   // stylelint-disable-next-line value-no-vendor-prefix
   cursor: -webkit-grab;
+  transition-property: width, height, margin;
```

Comment:
Can we transition scale instead? The pixel jogging from the width/height animation was pretty noticeable to be straight away. Based on its position on the track we can change the transform origin so it appears to scale to the right, center, or left.

Reply:
Did a quick inspector mock up and scale appears to work well for dual thumbs

![scale](https://user-images.githubusercontent.com/344839/73223355-a156d900-411a-11ea-80a6-250d5811fd91.gif)

Reply:
Great suggestion! Scale definitely looks a lot smoother

Reply:
Addressed with https://github.com/Shopify/polaris-react/pull/2655/commits/9c3c7430706d31cbbbe99c7d159d2054c0f07303